contracts/UniswapV3Staker.sol

```diff
@@ -148,7 +148,8 @@ contract UniswapV3Staker is IUniswapV3Staker, Multicall {
                 )
             );
 
-        int24 tickSpacing = IUniswapV3Pool(pool).tickSpacing();
+        int24 tickSpacing = factory.feeAmountTickSpacing(fee);
+        require(tickSpacing > 0, "staker: !fee");
```

Comment:
Would slightly change the string formatting here to be similar to rest of contract

```
require(tickSpacing > 0, 'UniswapV3Staker::createIncentiveWithMaxRange: !fee');
```